cedar-drt/set_env_vars.sh

```diff
@@ -1,6 +1,22 @@
+#!/bin/bash
+
+# Set JAVA_HOME
 if [ -z "${JAVA_HOME-}" ]; then
     # Idea from https://www.baeldung.com/find-java-home.
     export JAVA_HOME="$(java -XshowSettings:properties -version 2>&1 | sed -ne 's,^ *java\.home = ,,p')"
 fi
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH+$LD_LIBRARY_PATH:}$JAVA_HOME/lib/amd64/server
-export CLASSPATH="$(< ../cedar-dafny-java-wrapper/build/runtimeClasspath.txt):$(pwd)/../cedar-dafny-java-wrapper/build/libs/cedar-dafny-java-wrapper.jar"
+
+# Set LD_LIBRARY_PATH
+libjvm_dirs=($(find "$JAVA_HOME" -name 'libjvm.*'))
```

Comment:
This is not working for me when I follow the instructions to run DRT.  I get the usual error: `fuzz/target/x86_64-unknown-linux-gnu/release/rbac: error while loading shared libraries: libjvm.so: cannot open shared object file: No such file or directory`.  It looks like the problem is that compared to my version [here](https://github.com/cedar-policy/cedar-spec/pull/30/files/2187bd634789001de55cbfc4b62299f57f70d9db#diff-fe65b77103b1950854ab4df96bca59a7c3431a4106c43366f2e526d64c461606R12), you removed the `-printf '%h\n'`, so the command prints the path to `libjvm.so` instead of the parent directory, and `LD_LIBRARY_PATH` gets set (on my system) to `/usr/lib/jvm/java-11-amazon-corretto.x86_64/lib/server/libjvm.so`, which doesn't work because `LD_LIBRARY_PATH` is supposed to contain directories.

What was your motivation in making the change?  To make the script work on macOS?  If we want to make DRT actually work on macOS, I suspect we have a lot more work to do than this.  For one thing, I think we would need to set `DYLD_LIBRARY_PATH` instead of `LD_LIBRARY_PATH`.

Unless I'm missing something, I'd like to fix this code to output the parent directory again so it works on my system.  If you want, I can do that in a way that still runs successfully on macOS, for what that's worth.  If it were my decision, I'd be more inclined to remove this illusion of generality and revert the whole thing to my version.

(Was there discussion elsewhere that I missed?)

Aside: It looks like the reason the CI didn't catch the problem before the PR was merged is that `ci.yml` duplicates parts of the logic from `set_env_vars.sh` rather than calling it.  I don't know if that's worth changing.

Reply:
*sigh* Sorry, I should have tested my solution better. I'll file an issue shortly.

Yes, removing the `-printf` was to make the script work on macOS. I (incorrectly) assumed this wouldn't break anything else. The current script works in my local development environment (macOS).

Reply:
Also: Yes, `ci.yml` should call the `./set_env_vars.sh` and `./build.sh` scripts directly to avoid errors like this in the future. I (again incorrectly) assumed that my changes wouldn't break anything.

Reply:
Issue here: https://github.com/cedar-policy/cedar-spec/issues/41 @mattmccutchen-amazon can I assign this to you?

Reply:
I assigned #41 to myself.